client.go

```diff
@@ -280,122 +277,9 @@ func (c *Client) RemoveDistributedObjectListener(ctx context.Context, subscripti
-	opts := hazelcast.SQLOptions{}
-	opts.SetCursorBufferSize(1000)
-	p[ts.SetQueryTimeout(5*time.Second)
-	result, err := client.ExecSQLWithOptions(context.TODO(), q, opts, 1001, 35, "Jane Doe")
-	// handle the error
-	cnt, err := result.RowsAffected()
-	// handle the error
-	fmt.Printf("Affected rows: %d\n", cnt)
-
-Note that LastInsertId is not supported and at the moment AffectedRows always returns 0.
-*/
-func (c *Client) ExecSQLWithOptions(ctx context.Context, query string, opts SQLOptions, params ...interface{}) (driver.Result, error) {
-	var err error
-	ctx, err = updateContextWithOptions(ctx, opts)
-	if err != nil {
-		return nil, err
-	}
-	return c.db.ExecContext(ctx, query, params...)
-}
-
-/*
-QuerySQL runs the given SQL query on the member-side and returns a row iterator.
-This method is used for SQL queries that return rows, such as SELECT and SHOW MAPPINGS.
-Placeholders in the query is replaced by parameters.
-A placeholder is the question mark (?) character.
-For each placeholder, a corresponding parameter must exist.
-
-Example:
-
-	q :=`SELECT name, age FROM person WHERE age >= ?`
-	rows, err := client.QuerySQL(context.TODO(), q, 30)
-	// handle the error
-	defer rows.Close()
-	var name string
-	var age int
-	for rows.Next() {
-		err := rows.Scan(&name, &age)
-		// handle the error
-		fmt.Println(name, age)
-	}
-*/
-func (c *Client) QuerySQL(ctx context.Context, query string, params ...interface{}) (*sql.Rows, error) {
-	return c.db.QueryContext(ctx, query, params...)
-}
-
-/*
-QuerySQLWithOptions runs the given SQL query on the member-side and returns a row iterator.
-This method is used for SQL queries that return rows, such as SELECT and SHOW MAPPINGS.
-Placeholders in the query is replaced by parameters.
-A placeholder is the question mark (?) character.
-For each placeholder, a corresponding parameter must exist.
-This variant takes an SQLOptions parameter to specify advanced query options.
-
-Example:
-
-	q :=`SELECT name, age FROM person WHERE age >= ?`
-	opts := hazelcast.SQLOptions{}
-	opts.SetCursorBufferSize(1000)
-	rows, err := client.QuerySQLWithOptions(context.TODO(), q, opts, 30)
-	// handle the error
-	defer rows.Close()
-	var name string
-	var age int
-	for rows.Next() {
-		err := rows.Scan(&name, &age)
-		// handle the error
-		fmt.Println(name, age)
-	}
-
-*/
-func (c *Client) QuerySQLWithOptions(ctx context.Context, query string, opts SQLOptions, params ...interface{}) (*sql.Rows, error) {
-	var err error
-	ctx, err = updateContextWithOptions(ctx, opts)
-	if err != nil {
-		return nil, err
-	}
-	return c.db.QueryContext(ctx, query, params...)
-}
-
-// PrepareSQL creates a prepared statement with the given query.
-// See sql/driver documentation about how to use the prepared statement.
-func (c *Client) PrepareSQL(ctx context.Context, query string) (*sql.Stmt, error) {
-	return c.db.PrepareContext(ctx, query)
+// SQL returns a service to execute distributes SQL queries.
```

Comment:
It would be nice to have documentation is here as well.

Reply:
Thanks for catching that! We moved these usage examples to `example_test.go` under the `sql` package. They will be visible on Godoc.